src/main/java/org/broadinstitute/consent/http/db/DataAccessRequestDAO.java

```diff
@@ -87,6 +87,45 @@ WHERE dar.submission_date > now() - interval '1 year'
           """)
   List<DataAccessRequest> findApprovedDARsByDatasetId(@Bind("datasetId") Integer datasetId);
 
+  /**
+   * This query finds dataset ids submitted within the last year on dar-dataset combinations where
+   * the most recent vote is true. The query accomplishes this by creating a view that is a grouping
+   * of election reference ids and LAST vote in the group of final votes for all data access
+   * elections. We need to group them due to the case of multiple elections on a dar-dataset
+   * request. Election 1 may have been denied. Election 2 may have been approved. Election 3 may
+   * have been denied again. When we partition over the election reference id, we'll get all final
+   * votes. The `LAST_VALUE` function selects the last result in the partition, which would be
+   * `FALSE` in the example above. Outside the JOIN, we filter on groupings where the final vote
+   * value is `TRUE` so the denied election in the example would be filtered out.
+   *
+   * @param darReferenceId The DAR reference UUID
+   * @return List of approved Datasets for the DAR
+   */
+  @SqlQuery(
+      """
+              SELECT dd.dataset_id
```

Comment:
The google java formatter doesn't indent text blocks for us, so I don't think there's any rule about what the indent needs to be. I do think we should try to move the text to the left if possible, to make it easier to see the code without scrolling right. In TDR the convention is to align the text with the start of the `"""`.
```suggestion
  @SqlQuery(
      """
      SELECT dd.dataset_id
```